selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java

```diff
@@ -69,10 +68,16 @@ public void setup() throws Exception {
     projectExplorer.selectItem(PROJECT);
     notificationsPopupPanel.waitProgressPopupPanelClose();
 
-    projectExplorer.quickExpandWithJavaScript();
-    projectExplorer.openItemByPath(PATH_TO_APP_CLASS);
-    projectExplorer.scrollToItemByPath(PATH_TO_BOOK_IMPL_CLASS);
-    projectExplorer.openItemByPath(PATH_TO_BOOK_IMPL_CLASS);
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        PATH_FOR_EXPAND_APP_CLASS, APP_CLASS_NAME + ".java");
+
+    projectExplorer.openItemByPath(PROJECT + "/model");
```

Comment:
We can replace sequence of method **openItemByPath()** calls by one call of method **expandPathInProjectExplorer()**.